q2_longitudinal/_longitudinal.py

```diff
@@ -251,16 +258,6 @@ def _volatility(output_dir, metadata, state_column, individual_id_column,
     categorical.get_column(default_group_column)
     numeric.get_column(default_metric)
 
-    # Verify that columns specified are present in metadata (skipping the
```

Comment:
I dropped this validation because these cols are now validated [here](https://github.com/qiime2/q2-longitudinal/pull/95/files#diff-a858b54e976e73c6d4e1d349842393a2R258) and [here](https://github.com/qiime2/q2-longitudinal/pull/95/files#diff-a858b54e976e73c6d4e1d349842393a2R218). So I don't think this extra validation is important — I _think_ this is an artifact of ongoing development.